MinecraftClient/McTcpClient.cs

```diff
@@ -1558,6 +1566,61 @@ public void OnUpdateHealth(float health, int food)
                     ConsoleIO.WriteLogLine("You are dead. Type /respawn to respawn.");
                 }
             }
+            if (Settings.AutoEat)
```

Comment:
Same as with AutoAttack, `McTcpClient.cs` is quite large, so any automation should be moved to a ChatBot 🙂